### PR TITLE
update certification process

### PR DESCRIPTION
**Pull request by nkkashyap**
Signed-off-by: Neeraj Kumar Kashyap <nkashyap@in.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,17 +18,14 @@ With IBM Cloud Satellite, you can bring your own compute infrastructure that is
...
+   1. Include the subject line: `<storage-solution-name> integration`*. For example: `LocalVolume integration`. 
    1. Provide a high level description of the storage solution with a link to the user documentation or public repository.
 
-1. [Complete the onboarding process](https://ibm-10.gitbook.io/certified-for-cloud-pak-onboarding)
-
-1. [Complete the Certification Requirements checklist.](https://ibm-10.gitbook.io/certified-for-cloud-pak-onboarding/co-sell-1/tech-cert/architectural-questions)
-
-
-## Creating and testing your deployment
+## Registering with IBM Cloud
```
Comment (mssachan) [inline README.md]: Why same heading at 2 places? line 17 & 25?

File: README.md [hunk 2]
```diff
@@ -13,17 +18,14 @@ With IBM Cloud Satellite, you can bring your own compute infrastructure that is
 
 1. [Create an IBMid](https://cloud.ibm.com/registration).
 
-1. Email `contsto2@in.ibm.com`.
-   1. Include the subject line: `IBM Cloud Satellite Storage <storage-solution-name> Integration`*. For example: `IBM Cloud Satellite Storage LocalVolume Integration`. 
+1. Create an issue under [IBM Satellite Storage](https://github.com/IBM/ibm-satellite-storage/issues) repo.
+   1. Include the subject line: `<storage-solution-name> integration`*. For example: `LocalVolume integration`.
```
Comment (mssachan) [inline README.md:22]: remove extra `*`

File: README.md [hunk 3]
```diff
@@ -13,17 +18,14 @@ With IBM Cloud Satellite, you can bring your own compute infrastructure that is
...
    1. Provide a high level description of the storage solution with a link to the user documentation or public repository.
 
-1. [Complete the onboarding process](https://ibm-10.gitbook.io/certified-for-cloud-pak-onboarding)
-
-1. [Complete the Certification Requirements checklist.](https://ibm-10.gitbook.io/certified-for-cloud-pak-onboarding/co-sell-1/tech-cert/architectural-questions)
-
-
-## Creating and testing your deployment
+## Registering with IBM Cloud
+1. Collaborate with **`IBM Cloud Satellite Storage Team`** to setup your Satellite Location for devlopment and testing.
```
Comment (mssachan) [inline README.md]: `devlopment` ----> `development`

**Review by mssachan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nkkashyap) [reply README.md]: Fixed the heading

File: README.md (hunk 3 above)
Comment (nkkashyap) [reply README.md]: fixed the typo

File: README.md (hunk 2 above)
Comment (nkkashyap) [reply README.md:22]: remove the character

File: README.md [hunk 4]
```diff
@@ -1,30 +1,32 @@
...
+
+This repository is to collaborate with IBM Partners and Storage Solution Vendors to develop storage configuration templates for IBM Cloud Satellite.
 
 ## Overview
-With IBM Cloud Satellite, you can bring your own compute infrastructure that is in your on-premises data center, at other cloud providers, or in edge networks as a Satellite Location to IBM Cloud. Then, you use the capabilities of Satellite to run IBM Cloud services on this infrastructure, and consistently deploy, manage, and control your app workloads. For more information, see the IBM documentation.
+With IBM Cloud Satellite, you can bring your own compute infrastructure that is in your on-premises data-centre, at other cloud providers, or in edge networks as a Satellite Location to IBM Cloud. Then, you use the capabilities of Satellite to run IBM Cloud services on this infrastructure, and consistently deploy, manage, and control your app workloads. For more information, see the IBM documentation.
```
Comment (derekpoindexter) [inline README.md]:
```suggestion
With IBM Cloud Satellite, you can bring your own compute infrastructure that is in your on-premises data center, at other cloud providers, or in edge networks as a Satellite Location to IBM Cloud. Then, you use the capabilities of Satellite to run IBM Cloud services on this infrastructure, and consistently deploy, manage, and control your app workloads. For more information, see the IBM documentation.
```

File: README.md [hunk 5]
```diff
@@ -1,30 +1,32 @@
...
 - [IBM Cloud Satellite](https://www.ibm.com/cloud/satellite)
+- [ROSK in IBM Cloud Satellite](https://cloud.ibm.com/docs/satellite?topic=satellite-managed-services)
```
Comment (derekpoindexter) [inline README.md]:
```suggestion
- [Red Hat OpenShift on IBM Cloud Satellite](https://cloud.ibm.com/docs/satellite?topic=satellite-managed-services)
```

File: README.md [hunk 6]
```diff
@@ -1,30 +1,32 @@
...
 - [Getting started](https://cloud.ibm.com/docs/satellite?topic=satellite-getting-started)
+- [IBM Cloud Satellite Storage](https://cloud.ibm.com/docs/satellite?topic=satellite-sat-storage-template-ov)
+
 
 ## Satellite storage partner certification process
 ![Storage Template Registration Flow](./satellite-storage-registration-flow.png) 
 
-## Registering with IBM Cloud
+## Register with IBM Cloud
```
Comment (derekpoindexter) [inline README.md]:
```suggestion
## Registering with IBM Cloud
```

File: README.md [hunk 7]
```diff
@@ -1,30 +1,32 @@
...
-1. Email `contsto2@in.ibm.com`.
-   1. Include the subject line: `IBM Cloud Satellite Storage <storage-solution-name> Integration`*. For example: `IBM Cloud Satellite Storage LocalVolume Integration`. 
+1. Create an issue under [IBM Satellite Storage](https://github.com/IBM/ibm-satellite-storage/issues) repo.
+   1. Include the subject line: `<storage-solution-name> integration`*. For example: `LocalVolume integration`. 
    1. Provide a high level description of the storage solution with a link to the user documentation or public repository.
 
-1. [Complete the onboarding process](https://ibm-10.gitbook.io/certified-for-cloud-pak-onboarding)
-
-1. [Complete the Certification Requirements checklist.](https://ibm-10.gitbook.io/certified-for-cloud-pak-onboarding/co-sell-1/tech-cert/architectural-questions)
+## Development environment
```
Comment (derekpoindexter) [inline README.md]:
```suggestion
## Setting up your development environment
```

File: README.md [hunk 8]
```diff
@@ -1,30 +1,32 @@
...
+1. Collaborate with **`IBM Cloud Satellite Storage Team`** to setup your Satellite Location for development and testing.
```
Comment (derekpoindexter) [inline README.md]:
```suggestion
1. Collaborate with the **`IBM Cloud Satellite Storage Team`** to setup your Satellite Location for development and testing. 
```

File: README.md [hunk 9]
```diff
@@ -1,30 +1,32 @@
...
-
-1. [Complete the Certification Requirements checklist.](https://ibm-10.gitbook.io/certified-for-cloud-pak-onboarding/co-sell-1/tech-cert/architectural-questions)
+## Development environment
+1. Collaborate with **`IBM Cloud Satellite Storage Team`** to setup your Satellite Location for development and testing. 
 
-
-## Creating and testing your deployment
-
-1. Create a `deployment.yaml` file to deploy Kubernetes resources for the storage operator or driver in the development enviornment. Review the following example `deployment.yaml` to deploy the `local-volume` operator.
+## Test your storage solution with ROKS in IBM Cloud Satellite
```
Comment (derekpoindexter) [inline README.md]:
```suggestion
## Testing your storage solution in IBM Cloud Satellite 
```

File: README.md [hunk 10]
```diff
@@ -76,24 +78,31 @@ With IBM Cloud Satellite, you can bring your own compute infrastructure that is
...
+1. After you have verified your template in the development environment, update the issue. Include the following information in your issue and ensure that you remove any secrets from your files.
    1. Your `deployment.yaml` file.
    1. An example`pvc.yaml` file.
    1. An example `app.yaml`.
    1. The output of the `oc get sc` command that displays the storage classes your deployment creates.
    1. The output of the `oc get -n <namespace> pvc` command that displays the PVCs your deployment creates.
    1. The output of the `oc get -n <namespace> pods` command that displays the pods your deployment creates.
 
+
+## Solution validation and certification
```
Comment (derekpoindexter) [inline README.md]:
```suggestion
## Validating and certifying your solution
```

File: README.md [hunk 11]
```diff
@@ -76,24 +78,31 @@ With IBM Cloud Satellite, you can bring your own compute infrastructure that is
...
+
+IBM Team will collaborate with you to ensure your product works well with the IBM Cloud Paks. The intent of this certification is for our two companies to produce enterprise grade, best of breed, resilient, scalable solutions driving toward a consistent and maintainable deployed environment.
```
Comment (derekpoindexter) [inline README.md]:
```suggestion
The IBM storage team will collaborate with you to ensure your product works well with the IBM Cloud Paks. The intent of this certification is for our two companies to produce enterprise grade, best of breed, resilient, scalable solutions driving toward a consistent and maintainable deployed environment.
```

File: README.md [hunk 12]
```diff
@@ -140,38 +149,62 @@ In this example, the `devicepath` parameter is set dynamically. In a local stora
...
+         "regex": "^/dev/[a-zA-Z0-9]+$",
+         "max-length": "64",
+         "required": "true",
+         "category": "config",
+         "obfuscate": "false",
+         "type": "text"
       }
    ```
 
 #### Custom parameter reference
```
Comment (derekpoindexter) [inline README.md]:
```suggestion
## Understanding the Satellite storage template parameters
```

**Review by derekpoindexter — CHANGES_REQUESTED** [review]
commit suggestions

**Review by derekpoindexter — APPROVED** [review]

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**